aws-sso-instanceaccesscontrolattributeconfiguration/README.md

```diff
@@ -0,0 +1,12 @@
+# AWS::SSO::InstanceAccessControlAttributeConfiguration
```

Comment:
Update this file also

Reply:
Update to what specifically? It's the same layout as AWS::SSO::Assignment and AWS::SSO::PermissionSet .

Reply:
my bad, miss it with other md file